ci/scripts/audit.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -eux
 
 pushd zebedee
+    mvn clean install dependency:copy-dependencies -Dmaven.test.skip=true -Dossindex.skip=true
```

Comment:
@janderson2 this is a complete hack and really needs to be fixed properly but the audit build passes now for the first time at least 🚀

Reply:
@daiLlew I have opened a PR against this branch that I feel might tidy this up a bit but still fixes the build if you don't mind taking a look #373 .